src/libs/kata-sys-util/src/spec.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::path::PathBuf;
+
+use kata_types::container::ContainerType;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    /// unknow container type
+    #[error("unknow container type {0}")]
+    UnknowContainerType(String),
+    /// missing sandboxID
+    #[error("missing sandboxID")]
+    MissingSandboxID,
+    /// oci error
+    #[error("oci error")]
+    Oci(#[from] oci::Error),
+}
+
+const CRI_CONTAINER_TYPE_KEY_LIST: &[&str] = &[
+    // cri containerd
+    "io.kubernetes.cri.container-type",
+    // cri-o
+    "io.kubernetes.cri-o.ContainerType",
+    // docker shim
+    "io.kubernetes.docker.type",
+];
+
+const CRI_SANDBOX_NAME_KEY_LIST: &[&str] = &[
```

Comment:
Nit: `CRI_SANDBOX_NAME_KEY_LIST` -> `CRI_SANDBOX_ID_KEY_LIST` ?

Reply:
Thanks for your review, already updated.